utils/mock-data-loader/insert-mocks-gef.js

```diff
@@ -24,12 +24,14 @@ const insertMocks = async (token) => {
 
   const makerToken = await portalApi.loginViaPortal(BANK1_MAKER1);
 
-  for (const [item] of MOCKS.APPLICATION.entries()) {
-    item.userId = makerUserId;
-    const application = await api.createApplication(item, makerToken);
+  for (const item of MOCKS.APPLICATION.entries()) {
+    const deal = item[1];
```

Comment:
Why can't this just be `for (const deal of MOCKS.APPLICATION) {`?

Reply:
Have removed `entries`, was trying to keep it consistent.